src/familiar/freeFightFamiliar.ts

```diff
@@ -29,14 +47,25 @@ export function menu(includeExperienceFamiliars = true, canChooseMacro = false):
         familiar: $familiar`Crimbo Shrub`,
         expectedValue: 2500,
         leprechaunMultiplier: 0,
-        limit: "none",
+        limit: "special",
+      });
+    }
+
+    if (location.zone === "Dinseylandfill" && have($familiar`Space Jellyfish`)) {
```

Comment:
Should we consider other 100% combat stench zones too? May be hard to generalize to grab all such zones, but at the very least I feel like our default stesnch zone `The Haunted Kitchen` should be considered as well

Reply:
Haunted Kitchen isn't a stench zone, though.

Reply:
oh right my bad